static/app/views/replays/detail/page.tsx

```diff
@@ -118,4 +118,8 @@ const TimeContainer = styled('div')`
   align-items: center;
 `;
 
+const StyledTimeSince = styled(TimeSince)`
+  font-size: 14px;
```

Comment:
Is there a theme value for this?

P.theme.text.??

Reply:
same wavelength